### PR TITLE
replaced outdated cmdlet

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8821
source: https://devblogs.microsoft.com/scripting/should-i-use-cim-or-wmi-with-windows-powershell/

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

**Review by illfated — APPROVED** [review]
Looks good to me. Follows the PowerShell example criteria in https://docs.microsoft.com/powershell/module/cimcmdlets/get-ciminstance#example-3--get-instances-of-a-class-filtered-by-using-a-query .

File: windows/deployment/deploy-enterprise-licenses.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This topic describes how to deploy Windows 10 Enterprise E3 or E5 licenses with
 To determine if the computer has a firmware-embedded activation key, type the following command at an elevated Windows PowerShell prompt
 
 ```
```
Comment (illfated) [inline windows/deployment/deploy-enterprise-licenses.md:36]: Cosmetic detail: We could add "PowerShell" syntax highlighting here to make it even more obvious:
```suggestion
```PowerShell
```

File: windows/deployment/deploy-enterprise-licenses.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ This topic describes how to deploy Windows 10 Enterprise E3 or E5 licenses with
 To determine if the computer has a firmware-embedded activation key, type the following command at an elevated Windows PowerShell prompt
```
Comment (JohanFreelancer9) [inline windows/deployment/deploy-enterprise-licenses.md:34]:
```suggestion
To determine if the computer has a firmware-embedded activation key, type the following command at an elevated Windows PowerShell prompt:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review -One minor additional edit, please.

**Review by illfated — APPROVED** [review]
Looks very good to me.

**Comment by illfated** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**